site/content/contribute/more-info/webapp/redux/_index.md

```diff
@@ -1,28 +1,17 @@
 ---
 title: "Redux"
-heading: "Mattermost-Redux"
-description: "mattermost-redux is a library that contains logic for loading and working with data from the Mattermost server."
+heading: "Redux"
+description: "The Mattermost web app uses Redux as its state management library."
 date: "2020-04-01T12:00:00-04:00"
 weight: 8
 aliases:
   - /contribute/webapp/redux
 ---
 
-{{<newtabref href="https://github.com/mattermost/mattermost-webapp/tree/master/packages/mattermost-redux" title="mattermost-redux" >}} is a library that contains logic for loading and working with data from the Mattermost server. It's currently used primarily by the Mattermost web app and by some plugins, and it was previously used by the Mattermost mobile app. You can access the {{<newtabref href="https://github.com/mattermost/mattermost-webapp/tree/master/packages/mattermost-redux" title="repository here">}}, and discuss mattermost-redux in the {{<newtabref href="https://community.mattermost.com/core/channels/redux" title="Redux Mattermost community channel">}}.
+The Mattermost web app uses {{< newtabref href="https://redux.js.org/" title="Redux" >}} as its state management library. Its key features are a centralized data store for the entire app and a pattern for predictably modifying and displaying that spplication state. Notably, we're not using Redux Toolkit since a large portion of our Redux code predates its existence.
```

Comment:
I got rid of most of the references to `mattermost-redux` here since they don't really seem relevant any more. They're kind of left over from when it was a separate repo anyway